3485/3/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationController < Sinatra::Base
+  Dotenv.load
+  SESSION_SECRET = ENV['SESSION_SECRET']
+
+  configure do
+    enable :sessions
+    set :session_secret, SESSION_SECRET
+    set :views, 'app/views'
+    set :public_dir, 'public'
+    register Sinatra::Flash
+  end
+
+  not_found do
```

Comment:
```suggestion
  error ActiveRecord::RecordNotFound do
```
and add 
```ruby
set :show_exceptions, :after_handler
```
in the configure block